src/pages/Explorer/Category/ExplorerCategory.svelte

```diff
@@ -5,28 +5,50 @@
   import AssetSelector from '../Components/AssetSelector.svelte'
   import EmptyState from '../Components/EmptyState.svelte'
   import TypeSelector from '../Components/TypeSelector.svelte'
-  import { getItems, EntityType, RANGES, MenuItem } from '../const'
+  import { queryExplorerItems } from '../api'
+  import { currentUser } from '../store'
+  import { EntityType, RANGES, MenuItem } from '../const'
 
   export let activeMenu
   let items = []
   let range = ''
   let assets = []
-  let types = new Set(Object.values(EntityType))
+  let types = new Set(Object.values(EntityType).map((t) => t.key))
   let page = 1
+  let pages = 1
 
   function fetch() {
     const voted = activeMenu === MenuItem.LIKES
     const currentUserDataOnly = activeMenu === MenuItem.MY_CREATIONS
-    getItems({ voted, range, page, currentUserDataOnly }).then((res) => (items = res))
+    queryExplorerItems({ types: Array.from(types), voted, range, page, currentUserDataOnly }).then(
+      (res) => {
+        pages = res.pages
+        if (page === 1) {
+          items = res.items
+        } else {
+          items = [...items, ...res.items]
+        }
+      },
+    )
   }
 
   $: activeMenu, range, assets, types, page, fetch()
+
+  $: showEmpty =
+    (!$currentUser && activeMenu === MenuItem.MY_CREATIONS) ||
+    (items.length === 0 && activeMenu !== MenuItem.NEW)
+
+  function getAssets({ project, metricsJson }) {
+    const _metricsJson = Object.values(metricsJson).filter((m) => m.slug)
+    const assets = new Set([project, ..._metricsJson])
+    return Array.from(assets)
```

Comment:
`project` reference will not be the same as the one in `metricsJson`. Do the filtering inside the `.filter` function